scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-64_9x9.yaml

```diff
@@ -1,3 +1,54 @@
+QFN-64-1EP_9x9mm_P0.5mm_EP3.8x3.8xmm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://datasheet.lcsc.com/szlcsc/Realtek-Semicon-RTL8211EG-VB-CG_C69264.pdf (page 69)'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  #body_size_x_min: 8.9
+  body_size_x: 9.0
+  #body_size_x_max: 9.1
+  #body_size_y_min: 8.9
+  body_size_y: 9.0
+  #body_size_y_max: 9.1
+  lead_width_min: 0.18
+  lead_width_max: 0.30
+  lead_len_min: 0.30
+  lead_len_max: 0.50
```

Comment:
nominal sizes for both width and lengths are given so they must be included as well.
(I use the newest dimension entry style as it is the least amount of work)
```yaml
  lead_width: 0.15 .. 0.25 .. 0.3
  lead_len: 0.3 .. 0.4 .. 0.5
```